frontend/packages/ux-editor/src/utils/exportUtils.ts

```diff
@@ -0,0 +1,134 @@
+import type { ITextResource, ITextResources } from 'app-shared/types/global';
+import type {
+  ExportForm,
+  ExportFormComponent,
+  ExportFormPage,
+  ExportOption,
+  ExportTextResource,
+  ExportTextResourceValue,
+} from '../types/ExportForm';
+import type { IFormLayouts, IOption, ITextResourceBindings } from '../types/global';
+import type { KeyValuePairs } from 'app-shared/types/KeyValuePairs';
+import { internalLayoutToExternal } from '../converters/formLayoutConverters';
+import type { ExternalFormLayout } from 'app-shared/types/api';
+
+export const generateExportFormFormat = (
```

Comment:
This method might be changed by "Forsvaret"? 😊 We could consider to write is a class instead to isolate using private methods, and increase the readability. When using a class we do not need to pass the same arguments to multiple methods. We could just reference the data on the class instances instead. But I'm not sure if we should do that now or "Forsvaret" is contributing to this and changes will be made anyway.

Reply:
It might be changed, but in general I think we want to create an export format that is generic and not specific for Forsvaret. I have told them as well that they may need to transform the output on their side to get something that matches their needs exactly. 
At some point in the future, supporting some kind of mapping to customize the export format might be nice, but my thought is that we can cross that bridge when we get there 😇 
Writing it as a class might still be nice though 🤔